source/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -350,6 +352,36 @@ describe('When NOT in Chrome', {  browser: '!chrome' }, () => {
 })
 ```
 
+The following suite of tests will only execute when running in Firefox browser. It will overwrite the viewport resolution in one of the tests, and will merge any existing environment variables with the given ones.
```

Comment:
```suggestion
The following suite of tests will only execute when running in the Firefox browser. It will overwrite the viewport resolution in one of the tests, and will merge any current environment variables with the provided ones.
```